vernac/g_vernac.mlg

```diff
@@ -917,7 +918,7 @@ GRAMMAR EXTEND Gram
 END
 
 GRAMMAR EXTEND Gram
-  GLOBAL: command query_command class_rawexpr gallina_ext;
+  GLOBAL: command query_command class_rawexpr gallina_ext search_scope;
```

Comment:
Hi, you don't need to add `search_scope` here. You would need to add it to export it only if it were defined in the section.

Otherwise, I agree that `delimiter` is too generic. What about `scope_delimiter` then? If I'm proposing a change, it is because `search_scope` is indeed used as a search scope in `searchabout_query` but not really in `argument_spec`. So, it seems to me that `scope_delimiter` is more neutral and could be a good compromise.